<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Payment</class>
 <widget class="QDialog" name="Payment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDialog
{
 background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(45, 137, 162, 255), stop:1 rgba(255, 255, 255, 255))
}
QPushButton{
                          background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(76, 162, 45, 255), stop:1 rgba(255, 255, 255, 255));
                          border-style: inset;
                          border-style: outset;
                          border-radius: 10px;
                          border-color: beige;
                          font: bold 14p;
                          min-width: 10em;
                          padding: 6px;
}
QPushButton:pressed{
                          background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.164773 rgba(137, 152, 162, 255), stop:1 rgba(255, 255, 255, 255));
                          border-style: inset;
                          color: red;
                          border-style: outset;
                          border-width: 3px;
                          border-radius: 10px;
                          border-color: beige;
                          font: bold 14p;
                          min-width: 10em;
                          padding: 6px;
}
QTextBrowser
{
background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(156, 182, 162, 255), stop:1 rgba(117, 203, 228, 255));
	font: 75 italic 12pt &quot;MS Shell Dlg 2&quot;;
}
QLabel
{
font: 75 italic 8pt &quot;Magneto&quot;;
}
 QLineEdit {
     border: 2px solid gray;
     border-radius: 10px;
     padding: 0 8px;
     background: rgb(85, 170, 127);
 }
</string>
  </property>
  <widget class="QPushButton" name="Payment_pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>142</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Шукати</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>261</width>
     <height>49</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Будь ласка введіть номер особового рахунку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Pay">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>142</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Оплатити</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Сума</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
